LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2020 RDK Management
+   Copyright 2020 Metrological
```

Comment:
Please use Copyright [yyyy] [name of copyright owner] here as there is more than one party with an Apache license.